pkg/manager-nvme/manager.go

```diff
@@ -394,9 +394,20 @@ func (s *Storage) initialize() error {
 		"unallocatedBytes", s.unallocatedBytes,
 		"virtualizationManagement", s.virtManagementEnabled)
 
-	ns, err := s.device.IdentifyNamespace(CommonNamespaceIdentifier)
-	if err != nil {
-		return fmt.Errorf("Initialize Storage %s: Failed to identify common namespace: Error: %w", s.id, err)
+	var ns *nvme.IdNs
+	identifySuccess := false
+	for retryCount := 0; !identifySuccess; retryCount++ {
+		var err error
+		ns, err = s.device.IdentifyNamespace(CommonNamespaceIdentifier)
+		if err != nil {
+			if retryCount >= 2 {
```

Comment:
Any reason this should be configurable?

Reply:
Yes it should be configurable.

Reply:
Leaving this alone for now to address other issues.